.claude/agents/test-runner.md

```
@@ -0,0 +1,69 @@
+rebuilt. `-Dit.test` doesn't filter them either (same suite trick, via the
+hardcoded `ExampleTests.java`). Don't hand-roll this — use the existing skill:
+```
+.claude/skills/run-example-test/run-example-test.sh [-b] <TestClassName>
+```
+Pass `-b` (or delete `distribution/target/*.zip`) if `core` or another upstream
+module changed since the last build.
+
+## 2. If a test run fails
+Tests bind fixed ports, mainly 2000, 3000, 7007 but also others. Before concluding it's a real
+regression, check whether something else already holds one of those ports:
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)